### PR TITLE
Added Persian language

### PR DESCRIPTION
**Pull request by masihtak**

### CODE REVIEW

**Comment by lumaxis** [comment]
Hey @masihtak, thanks for the contribution! :blush: Have you tried if this localization works well with all the elements of our SDK and that everything is still displayed correctly? I don't think we have any right-to-left languages in our current set of translations yet so I'd like to be sure everything works well before merging this.

**Comment by TroubleMakerBen** [comment]
Hi @masihtak,
We'll review this soon and hope to include it in 4.1.1.

**Comment by TroubleMakerBen** [comment]
I'm terribly sorry but we have to postpone this. We're hoping to have it ready for 4.1.2.

File: Resources/fa.lproj/HockeySDK.strings [hunk 1]
```diff
@@ -0,0 +1,276 @@
...
+ */
+
+
+/* General */
+
+/* For dialogs yes buttons */
+"HockeyYes" = "بله";
+
+/* For dialogs no buttons */
+"HockeyNo" = "نه";
```
Comment (MehrdadMzfr) [inline Resources/fa.lproj/HockeySDK.strings]: should be:
خیر

File: Resources/fa.lproj/HockeySDK.strings [hunk 2]
```diff
@@ -0,0 +1,276 @@
...
+/* Replacement for app name, if it could not be detected */
+"HockeyAppNamePlaceholder" = "این برنامه";
+
+/* Crash */
+
+
+/* Crash dialog */
+
+/* Title showing in the alert box when crash report data has been found */
+"CrashDataFoundTitle" = "%@ ترک غیر منتظره";
```
Comment (MehrdadMzfr) [inline Resources/fa.lproj/HockeySDK.strings]: "ترک غیر منتظره %@";

**Comment by TroubleMakerBen** [comment]
Made small changes and created new PR for this #369

**Pull request closed without merging**